src/main/java/uk/gov/hmcts/darts/courthouse/controller/CourthousesController.java

```diff
@@ -77,9 +75,10 @@ public ResponseEntity<List<ExtendedCourthouse>> courthousesGet() {
     }
 
     @Override
-    public ResponseEntity<ExtendedCourthouse> courthousesPost(Courthouse courthouse) {
-        CourthouseEntity addedCourtHouse = courthouseService.addCourtHouse(courthouse);
-        ExtendedCourthouse extendedCourthouse = courthouseMapper.mapFromEntityToExtendedCourthouse(addedCourtHouse);
+    @SecurityRequirement(name = SECURITY_SCHEMES_BEARER_AUTH)
+    @Authorisation(contextId = ANY_ENTITY_ID, globalAccessSecurityRoles = {SUPER_ADMIN, SUPER_USER})
```

Comment:
I don't believe this create courthouse endpoint needs `SUPER_USER` - please remove
Keeping `SUPER_ADMIN`

Reply:
Spot on, removed.